tests/test_run.py

```diff
@@ -32,3 +32,17 @@ def test_run_command_grab_output_as_string(tmpdir):
     assert "Hello1" in output
     assert "Hello2" in output
     assert "Hello3" in output
+
+
+def test_run_invalid_output():
+    """Checks that invalid unicode bytestrings are handled correctly"""
+    invalid_byte_sequence = b'\x80\x81'
+    output = run.get_actual_output(invalid_byte_sequence)
+    assert invalid_byte_sequence in output
+
+
+def test_run_invalid_output_str():
+    """ Checks that ouputs that are already strings are handled correctly"""
+    randomString = str("Hello")
```

Comment:
Everything else looks good, but this should just be `string = "hello"` instead of using `str` unless there is a valid reason.

Reply:
Nope your right, recommitting now.